### ISSUE TITLE
create ApiLogicServer Slack

### ISSUE THREAD

**Issue by mingfang** [valhuber/ApiLogicServer#52]
It would be great to have something like Slack for discussions around ApiLogicServer.

**Comment by valhuber** [comment]
Good idea, and... it exists: **API Logic Server** is the workspace name

**Comment by mingfang** [comment]
That's great.
How can we join that workspace?

**Comment by valhuber** [comment]
I am new to Slack, but try joining that workspace

**Comment by valhuber** [comment]
@mingfang here is the invite link, come say hello! https://join.slack.com/t/apilogicserver/shared_invite/zt-1k36g88zp-olEy2Two39G7R~WXutzGDw
